src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/actions/GenerateTestsUtils.kt

```diff
@@ -122,6 +187,30 @@ private fun validateClass(psiClass: PsiClass): Boolean {
     return !psiClass.isEnum && psiClass !is PsiAnonymousClass
 }
 
+/**
+ * Checks if the selected line contains (part of) a statement and is a valid line to be selected for EvoSuite.
+ * Namely, the line is within a method, it is not blank and contains (part of) a statement.
+ *
+ * @param selectedLine selected line number
+ * @param psiMethod surrounding PSI method
+ * @param psiFile containing PSI file
+ * @return true if the line is valid, false otherwise
+ */
+private fun validateLine(selectedLine: Int, psiMethod: PsiMethod, psiFile: PsiFile): Boolean {
```

Comment:
Same for this one

Reply:
Same comment as above. Also, it would defeat the purpose of GenerateTestsUtils class. I think that such business logic has to be separated into a separate class, and action classes have to be responsible for giving (showing) the option to generate tests to the user (when appropriate), and react to the action